proto/executor.proto

```diff
@@ -154,7 +154,7 @@ message ANNQueryInfo {
 
 	// as we remove the vector column in table_scan, we need to save the vector column type to indicate which vector type(nullable or not) 
 	// we need to read when index is not built.
-	optional FieldType vector_column_type = 22 [(gogoproto.nullable) = true];
+	optional ColumnInfo column_info = 22 [(gogoproto.nullable) = false];
```

Comment:
Please move this after L138 because they are together and this field is already present.